input.go

```diff
@@ -4,9 +4,9 @@
 
 package engi
 
-type Action int
-type Key int
-type Modifier int
+type Action 	int
+type Key 	int
+type Modifier 	int
```

Comment:
I have seen it, I think ...  in my IDE (`WebStorm` / `IDEA` (with [Go plugin](https://github.com/go-lang-plugin-org/go-lang-idea-plugin)) - but this gets reverted when I run `gofmt`.